docs/correlate-logs-traces.md

```diff
@@ -0,0 +1,8 @@
+# Correlating traces with logs
+
+TODO:
+
+- Attributes that are injected
+- List of supported logging librares
+- Sample formatting and formatting considerations
+- How to enable/disable trace injection into logs
```

Comment:
@Rauno56 TODO

Reply:
I think we've talked about all of those by now?